### PR TITLE
CPOL 58 : adding titles to wizard.

### PR DESCRIPTION
**Pull request by spinder**
Not sure if banner titles work here. Perhaps a lighter gray color? 50% width?

### CODE REVIEW

File: src/components/Policy/WizardSteps/ReviewStep.tsx [hunk 1]
```diff
@@ -21,6 +22,7 @@ const ReviewStep: React.FunctionComponent = () => {
     return (
         <>
             <Form>
+                <Label>{Messages.wizardReview}</Label>
```
Comment (josejulio) [inline src/components/Policy/WizardSteps/ReviewStep.tsx]: I think you can use the [Title](https://patternfly-react.surge.sh/documentation/react/components/title) component, IMO the `xl Title` size would be good.

File: src/components/Policy/WizardSteps/ActionsStep.tsx [hunk 2]
```diff
@@ -20,6 +21,7 @@ const ActionsStep = () => {
 
     return (
         <Form>
+            <Label>{Messages.wizardActions}</Label>
```
Comment (josejulio) [inline src/components/Policy/WizardSteps/ActionsStep.tsx]: I'm sure the linter will complain about this line:

```suggestion
            <Label>{ Messages.wizardActions }</Label>
```
You can check by running `yarn lint`

File: src/components/Policy/WizardSteps/ReviewStep.tsx (hunk 1 above)
Comment (spinder) [reply src/components/Policy/WizardSteps/ReviewStep.tsx]: Done. Looks better.

**Comment by josejulio** [comment]
I'm promoting the Draft to PR.
In the next PRs, could you add a gif or screenshot of the changes?

**Comment by josejulio** [comment]
LGTM

**Pull request merged**